### ISSUE TITLE
USER ISSUE: Flying high in the air

### ISSUE THREAD

**Issue by Flamefire** [StrangeLoopGames/EcoIssues#22439]
\
**User Id: (Not logged user)**
**Version:** 0.9.3.5 beta release-231

**Steps to Reproduce:**
No idea

**Expected behavior:**

**Actual behavior:**
Without any reason I'm shot high in the air and never come back

**Do you have mods installed? Does the issue happen when no mods are installed?:**
no

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by TwixterZ** [comment]
@Flamefire, thank you for raising this and sorry that this happen

- Does this issue happens often or is it a one time issue?
- There a known issue where a fallen tree known to shoot player to space, do you recall this is what had happen?
- Would be helpful if you can give us a video when this happens for us to pinpoint this issue.

**Comment by Flamefire** [comment]
This happens every now and then. Often it is just to low altitude so you gracefully fall back. But on time it was ~1500m and there was no way to get back at all and I had to use admin commands (kinda undocumented) to continue playing at all.

Any movable entity may trigger that. E.g. stepping on a cart in a narrow place triggered it too for us. Can't remember what we did exactly and not to keen on repeating that experience ;)
